packages/transform-to-vanilla/src/transform-object/index.ts

```diff
@@ -0,0 +1,48 @@
+import { simplyImportant } from "@/transform-values/simply-important";
+import type { StyleRule } from "@vanilla-extract/css";
+import type {
+  CSSRule,
+  CSSRuleKey,
+  CSSRuleValue,
+  StyleRuleValue
+} from "@/types/style-rule";
+
+// == Interface ================================================================
+export function transformStyle(style: CSSRule) {
+  const result: {
+    // key in StyleRuleKey occur a error
+    // Type '{}' is missing the following properties from type
+    // '{ accentColor: StyleRuleValue; alignContent: StyleRuleValue; alignItems: StyleRuleValue; alignSelf: StyleRuleValue; ... 904 more ...;
+    //  vars: StyleRuleValue; }': accentColor, alignContent, alignItems, alignSelf, and 905 more.ts(2740)
```

Comment:
You mean this function cause error for now?

Reply:
No, because if you use `const result = { [key in StyleRuleKey]: StyleRuleValue; }`(`result: StyleRule`), you'll get an error and have to use